doc/fern-renderer-nerdfont.txt

```diff
@@ -61,6 +61,36 @@ VARIABLE				*fern-renderer-nerdfont-variable*
 	Enabling this option may affect performance.
 	Default: 0
 
+*g:fern#renderer#nerdfont#root_leading*
+	Add a lead (padding) in front of the root level nodes.
+        By default uses *g:fern#renderer#nerdfont#leading* when
+        set to 1, when it's 0 no padding is added.
+
+        When it's a |String|, that value will be used instead
+        in front of the root level nodes.
+
+	Default: 1
+
+        Example, if root_leading set to 1 nodes will look like
+        the following (shown with *'number'* option enabled):
+>
+    1│ root
+    2│  file1.txt
+    3│  file2.txt
+<
+        With root_leading set to 0:
+>
+    1│root
+    2│ file1.txt
+    3│ file2.txt
+<
+        With root_leading set to "@  ":
+>
+    1│@  root
+    2│@   file1.txt
+    3│@   file2.txt
+<
+
```

Comment:
The option must follow the rule of `g:fern#renderer#nerdfont#leading`, mean that setting `1` should be avoided.

Reply:
Yes, by default they don't have to explicitly set to `1` and it will use `g:fern#renderer#nerdfont#leading` value.

Unless you mean to omit in the docs?

Reply:
No, I ment that `let g:fern#renderer#nerdfont#root_leading = 1` should be error because `let g:fern#renderer#nerdfont#leading = 1` cause error.

Reply:
Well, I'm asking you to remove the following feature.

> when g:fern#renderer#nerdfont#root_leading is set to 1 then it will use whatever g:fern#renderer#nerdfont#leading value is.

Because the default value could be specified like

```vim
let g:fern#renderer#nerdfont#leading = get(g:, 'fern#renderer#nerdfont#leading', ' ')
let g:fern#renderer#nerdfont#root_leading = get(g:, 'fern#renderer#nerdfont#root_leading', g:fern#renderer#nerdfont#leading)
```